src/exercises/MappingThroughAListAndRenderingCustomComponent.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+export default function MappingThroughAListAndRenderingCustomComponent() {
+  return (
+    <div>
+      {[
+        {
+          name: "dog",
+          icon: "🐶",
+        },
+        {
+          name: "cat",
+          icon: "🐱",
+        },
+        {
+          name: "chicken",
+          icon: "🐔",
+        },
+        {
+          name: "cow",
+          icon: "🐮",
+        },
+        {
+          name: "sheep",
+          icon: "🛌",
+        },
+        {
+          name: "horse",
+          icon: "🏇",
+        },
+      ].map((e) => {
```

Comment:
-can you create variable for this array 
-and while you are mapping you need to provide good name for the parameter it can be event